packages/lms/src/lib/components/navigation/CourseNav.svelte

```diff
@@ -31,22 +36,27 @@
 		font-weight: 600rem;
```

Comment:
`font-weight: 600rem`
Seems a bit excessive, no?

Reply:
MORE COWBELL

Reply:
![cowbell](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExMTY5Yjc3MGIwN2FmMDdhZmE0ZDk2ZTRhOGM2YzM4MWQ5OTc0MmMxNyZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/l0HlBwsIWjIgEQXMk/giphy.gif)

Reply:
knocked it down to 500